sections/preliminaries.tex

```diff
@@ -0,0 +1,136 @@
+\section{Preliminaries}\label{sec:preliminaries}
+This section covers previous work within topic modelling that will be expanded upon in this paper.
+
```

Comment:
```suggestion
These are all \gls{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have an influence on both word distributions and other metadata

```